requirements-dev.txt

```diff
@@ -2,3 +2,4 @@ pytest
 pytest-cov
 black
 mypy
+python-semantic-release
```

Comment:
shouldn't `requirements*.txt` files be incorporated into `setup.py` and removed from the repo?

Reply:
I kind a like requirements files and easy separation between runtime and development requirements.

Reply:
Having both requirements file and setup.py means that dependencies have to be tracked/maintained in several places which is error-prone. During the development process module is usually installed with `pip install -e .[dev]` (see [setup.py section](https://github.com/F-Secure/fliprate_actions/pull/17/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R46)) during usage with `pip install -e .`